es55x/src/main/scala/tech/beshu/ror/es/ReadonlyRestPlugin.scala

```diff
@@ -160,7 +162,8 @@ class ReadonlyRestPlugin(s: Settings,
                                indexNameExpressionResolver: IndexNameExpressionResolver,
                                nodesInCluster: Supplier[DiscoveryNodes]): util.List[RestHandler] = {
     List[RestHandler](
-      new RestRRAdminAction(settings, restController)
+      new RestRRAdminAction(settings, restController),
+      new RestRRConfigAction(settings, restController, nodesInCluster),
```

Comment:
I'm not sure if I understand correctly, but IMO:
* endpoint for frontend should be exposed by AdminRestApi backed by finch (ROR Proxy will take advantage if it)
* internode communication related to config checking should be done as ES action (this won't work for Proxy, but atm it doesn't need it)

Reply:
I think that in current implementation endpoint

`val provideRorConfigPath: Path = Path.create(NonEmptyList.of("_readonlyrest", "admin", "config", "load"))`

is not handled by Finch

Reply:
It isn't. I completely missed Finch :/ I just read about finch. If finch is used, as good abstraction for http service to reuse it in different esXXx modules, it's not enough abstraction. `io.finch.Endpoint` is defined as `=1=>`, `=2=>`, and `=3=>`. In my case, `Endpoint` concept is nested for internode communication, and I used here ES action transport implementation. We'll talk.